<!--

throw → used to generate (raise) an error.
try–catch–finally → used to handle that error.
throws →  does not exist in JavaScript.
JavaScript does not support throws.
We only use throw to raise exceptions.
-->
<html>
<head>
  <title>Throw Example</title>
</head>
<body>
  <script>
    function checkAge(age) {
      if (age < 18) {
        throw "Age must be 18 or older"; // throwing a string error
      }
      return "Access granted";
    }

    try {
      let result = checkAge(25); // will throw error
      document.write(result);
    } catch (err) {
      document.write("Caught Exception Ram: " + err);
    } finally {
      document.write("<br>End of program.");
    }
  </script>
</body>
</html>
